### ISSUE TITLE
Training wont start again [Google Colab]

### ISSUE THREAD

**Issue by MarkupMark** [voicepaw/so-vits-svc-fork#409]
I trained a model with 100 samples under 12s, I did 1000+ epochs of training on google colab, then i saved my project. Now i can't train the model anymore, the training phase lenght doesn't get over 10 seconds, it just shows the tensorboard graph like this, without giving any error message.
![Screenshot 2023-04-19 alle 22 28 48](https://user-images.githubusercontent.com/50554987/233192569-feb96724-c180-4858-89dc-6170edf7256d.png)

The model actually works but it needs more steps for being actually usable.

Anything useful is appreciated, thanks

**Comment by fran0215** [comment]
I have the same problem. I am unable to continue training the model.

**Comment by billythemusical** [comment]
Did you reload the page or get disconnected?  Google Colab may have deleted some of the runtime files you need to make things run properly.

**Comment by fran0215** [comment]
the network connection was disconnected. How would I pause the training and be able to continue later without errors?

**Comment by fran0215** [comment]
![image](https://user-images.githubusercontent.com/63940553/235267305-47e5e6c2-29c4-4372-b339-5a8524f8df5d.png)

also for this message the training is canceled.

**Comment by billythemusical** [comment]
As far as I know, you can't pause the training and come back later. If you get disconnected before the training is completed, you'll have to start the notebook over again at the top because all the files and variables that are used in the training are no longer available.

Once the training is completed, the notebook tries to save it to your Google Drive so you can re-use it, which is nice. I just don't know of a way to do the full 10000 Epochs without Google Colab bumping me off, either from not clicking the "Are you still there?" box or just from generally being kicked off. I've only done 1000 Epochs and it took 3+ hours on the free tier.

**Comment by fran0215** [comment]
I reached 2000 epoch in 4 hours for 2 days. I guess I've been fine then.
what I did when the training was canceled was in the training block add "re" to "%load_ext tensorboard" looking like this "%reload_ext tensorboard". It has worked after a few tries.

**Comment by billythemusical** [comment]
Yeah, I just ignore that warning. It doesn't crash even if it's already loaded.
